dynamiqs/mesolve/mesolve.py

```diff
@@ -27,118 +27,122 @@ def mesolve(
 ) -> Result:
     r"""Solve the Lindblad master equation.
 
-    Evolve the density matrix $\rho(t)$ from an initial state $\rho(t=0) = \rho_0$
-    according to the Lindblad master equation using a given Hamiltonian $H(t)$ and a
-    list of jump operators $\{L_k\}$. The Lindblad master equation is given by
-
+    This function computes the evolution of the density matrix $\rho_t$, starting
+    from an initial state $\rho_0$ at time $t=0$, according to the Lindblad master
+    equation:
     $$
-        \frac{d\rho}{dt} = -i[H, \rho] + \sum_k \left(L_k \rho L_k^\dagger -
-        \frac{1}{2} \left \{L_k^\dagger L_k, \rho\right\}\right).
+        \frac{\mathrm{d}\rho_t}{\mathrm{d}t} =-i[H_t, \rho_t]
+        + \sum_{k=1}^N \left(
+            L_k \rho_t L_k^\dagger
+            - \frac{1}{2} L_k^\dagger L_k \rho_t
+            - \frac{1}{2} \rho_t L_k^\dagger L_k
+        \right),
     $$
+    where $H_t$ is the system's Hamiltonian at time $t$ and $\{L_k\}$ is a collection of
+    $N$ jump operators.
```

Comment:
Right, but then we should do an implicit summation, I thought it may be a little confusing for newcomers/non-mathematical guys, but I'm fine with doing this change.

$$
        \frac{\mathrm{d}\rho_t}{\mathrm{d}t} =-i[H_t, \rho_t]
        + \sum_{k=1}^N \left(
            L_k \rho_t L_k^\dagger
            - \frac{1}{2} L_k^\dagger L_k \rho_t
            - \frac{1}{2} \rho_t L_k^\dagger L_k
        \right)
    $$

$$
        \frac{\mathrm{d}\rho_t}{\mathrm{d}t} =-i[H_t, \rho_t]
        + \sum_k \left(
            L_k \rho_t L_k^\dagger
            - \frac{1}{2} L_k^\dagger L_k \rho_t
            - \frac{1}{2} \rho_t L_k^\dagger L_k
        \right)
    $$

Reply:
No I would keep the $N$ in the equation, I just didn't think it was necessary in the text precisely because it was in the equation. Maybe you could use $\{L_k\}_{k\in[1,N]}$ if you want completeness.

Reply:
I'm fine with just removing it from the text 👍